### PR TITLE
Follow-up for SettingsNotice

### PR DESCRIPTION
**Pull request by aaemnnosttv**
## Summary

Addresses issue #3251

## Relevant technical choices

* Decouples common styles from type-specific styles
* Simplifies markup into more common classes between line-variant components, making the variable part more specific to the body of the notice
* Move svg sizing to props on `Icon`
* Fixes some styles for sizing of notice line-height with icon
* Updated icons to inherit color rather than have a hardcoded value (like others)

**Fixes alignment/sizing of icon relative to notice text**
image

**Fixes styling of borders and text in inner Select components**
image

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/components/SettingsNotice/SettingsNotice.js [hunk 1]
```diff
@@ -19,24 +19,41 @@
...
-		<div className={ `googlesitekit-settings-notice googlesitekit-settings-notice--${ type }` } >
-			{ !! children && <SettingsNoticeMultiRow { ...props } /> }
-			{ ! children && (
-				<SettingsNoticeSingleRow { ...props } />
+		<div
+			className={ classnames(
+				'googlesitekit-settings-notice',
+				`googlesitekit-settings-notice--${ type }`,
+				{ 'googlesitekit-settings-notice--single-row': ! children },
+				{ 'googlesitekit-settings-notice--multi-row': children },
```
Comment (eugene-manuilov) [inline assets/js/components/SettingsNotice/SettingsNotice.js:47]:
```suggestion
				{
					'googlesitekit-settings-notice--single-row': ! children,
					'googlesitekit-settings-notice--multi-row': children,
				},
```

File: assets/js/components/SettingsNotice/SettingsNoticeMultiRow.js [hunk 2]
```diff
@@ -17,54 +17,43 @@
...
  */
-import PropTypes from 'prop-types';
+import { Fragment } from '@wordpress/element';
 
 /**
- * Internal dependencies
+ * External dependencies
  */
-import { TYPE_WARNING, TYPE_INFO, TYPE_SUGGESTION, getIconFromType } from './utils';
+import PropTypes from 'prop-types';
```
Comment (eugene-manuilov) [inline assets/js/components/SettingsNotice/SettingsNoticeMultiRow.js:27]: @aaemnnosttv external dependencies should be defined higher than wordpress deps.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Awesome! Thanks, @aaemnnosttv. Found a few tiny issues. Could you please take a look at it?

File: assets/js/components/SettingsNotice/SettingsNoticeMultiRow.js (hunk 2 above)
Comment (aaemnnosttv) [reply assets/js/components/SettingsNotice/SettingsNoticeMultiRow.js:27]: Oops 🤦

**Review by eugene-manuilov — APPROVED** [review]
LGTM!

**Pull request merged**